src/test/java/uk/gov/justice/digital/delius/integration/secure/OffendersResource_getOffenderNsisByCrn.java

```diff
@@ -127,30 +148,30 @@ public void getGetOffenderNsisByCrnAndConvictionIdLimitCode() {
 
     @Test
     public void getGetOffenderNsisByCrnAndUnknownConvictionId_Returns404() {
```

Comment:
I'm confused how this test used to work, doesn't this show you did get a 404 for a unknown eventId?

Reply:
Ignore me, that wasn't the fix